CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * Your contribution here.
 
+#### 0.12.3 (2020/8/17)
```

Comment:
This is duplicating the above line, just add a 1-liner for the change, remove this line and leave `#### 0.12.3 (Next)`